app/services/cocina_support.rb

```diff
@@ -88,6 +88,10 @@ def self.abstract_for(cocina_object:)
     cocina_object.description.note.find { |note| note.type == 'abstract' }&.value
   end
 
+  def self.citation_for(cocina_object:)
+    cocina_object.description.note.find { |note| note.type == 'preferred citation' }&.value
```

Comment:
Do we not use the label (from the `CocinaDescriptionSupport` change above) when we map back from Cocina?